### PR TITLE
Seo preflight best practices tool tip

### PR DESCRIPTION
**Pull request by rayyank10**
incorporate a 'Best Practices' tooltip in the Milo Preflight interface, aimed at providing authors with valuable insights regarding optimal SEO techniques.
eg : Keep the title between 50~70 characters. under title icon in milo preflight 
Ticket : https://jira.corp.adobe.com/browse/MWPW-129722
**Test URLs:**
- Before: https://main--milo--adobecom.hlx.page/?martech=off
- After: https://seobestpractices--milo--adobecom.hlx.page/?martech=off

### CODE REVIEW

**Comment by mokimo** [comment]
@rayyank10 is there any ticket that you could link in the desciption?

**Comment by mokimo** [comment]
This introduces two scroll bars on chrome
Screenshot 2023-06-01 at 11 45 39

**Comment by rayyank10** [comment]
> @rayyank10 is there any ticket that you could link in the desciption?

added the ticket

File: libs/blocks/preflight/panels/seo.js [hunk 1]
```diff
@@ -53,7 +56,7 @@ async function checkCanon() {
   const canon = document.querySelector("link[rel='canonical']");
   const result = { ...canonResult.value };
   const { href } = canon;
-
+  result.best = 'Self-reference canonical is preferred as this informs search engines what version of the page you want to show up in SERP.'
```
Comment (mokimo) [inline libs/blocks/preflight/panels/seo.js:59]: This sentence is complex, you could try to rephrase it

File: libs/blocks/preflight/panels/seo.js [hunk 2]
```diff
@@ -2,21 +2,23 @@ import { html, signal, useEffect } from '../../../deps/htm-preact.js';
...
+const canonResult = signal({ icon: DEF_ICON, title: 'Canonical', description: DEF_DESC, best: DEF_BEST });
+const descResult = signal({ icon: DEF_ICON, title: 'Meta description', description: DEF_DESC, best: DEF_BEST });
+const bodyResult = signal({ icon: DEF_ICON, title: 'Body size', description: DEF_DESC, best: DEF_BEST });
+const loremResult = signal({ icon: DEF_ICON, title: 'Lorem Ipsum', description: DEF_DESC, best: DEF_BEST });
+const linksResult = signal({ icon: DEF_ICON, title: 'Links', description: DEF_DESC, best: DEF_BEST });
 
 function checkH1s() {
   const h1s = document.querySelectorAll('h1');
   const result = { ...h1Result.value };
+  result.best = 'Maintaining a single H1 header on a page is optimal as it helps prevent confusion.'
```
[the remark this answers is not in the document]
Comment (mokimo) [reply libs/blocks/preflight/panels/seo.js:21]: Agree, `hint` or `bestPractice` would provide more clarity. Or even a mix such as `bestPracticeHint`

**Comment by auniverseaway** [comment]
1. I think this UX needs improvement if we're going to ship this.
2. I prefer the progressive disclosure we had before (which would make this PR kinda redundant)... Why are we telling people what the requirements are if they already met them? "You should have 10 fingers and 10 toes." "Uh, thanks."
3. If we _are_ going to ship something like this, I would probably make this more authorable and pull these text strings from a JSON file. At the very least, we should do that for the existing guidance prompts.

**Comment by auniverseaway** [comment]
> This introduces two scroll bars on chrome Screenshot 2023-06-01 at 11 45 39

I fixed this in a previous PR. I updated his branch and it should be fixed.

**Review by chrischrischris — COMMENTED** [review]
Be sure to run eslint as I see some lines without `;` which it should have complained about.

**Comment by rayyank10** [comment]
> 1. I think this UX needs improvement if we're going to ship this.
> 2. I prefer the progressive disclosure we had before (which would make this PR kinda redundant)... Why are we telling people what the requirements are if they already met them? "You should have 10 fingers and 10 toes." "Uh, thanks."
> 3. If we _are_ going to ship something like this, I would probably make this more authorable and pull these text strings from a JSON file. At the very least, we should do that for the existing guidance prompts.


I think logic is if an author gets an error message telling them their title is not long enough they are not just guessing how much longer it needs to be they can just hover the tooltip and get an idea on what they need to do.  Can definitely improve the ux. let me talk with the team and come back around.

**Pull request closed without merging**